build.rs

```diff
@@ -634,11 +632,13 @@ where
 
 fn run_command(mut cmd: Command) {
     println!("running {:?}", cmd);
-    let status = cmd.status().unwrap_or_else(|e| {
+    let output = cmd.output().unwrap_or_else(|e| {
```

Comment:
Why did you switch from `status()` to `output()`?

Reply:
I think that was in there from when I was debugging why yasm was failing, I'll revert.